src/System.Private.ServiceModel/tests/Scenarios/Client/ClientBase/DuplexClientBaseTests.cs

```diff
@@ -11,10 +11,24 @@
 
 public class DuplexClientBaseTests : ConditionalWcfTest
 {
+#if FULLXUNIT_NOTSUPPORTED
+    [Fact]
+#else
     [ConditionalFact(nameof(Root_Certificate_Installed))]
+#endif
     [OuterLoop]
     public static void DuplexClientBaseOfT_OverHttp_Call_Throws_InvalidOperation()
```

Comment:
Interestingly we don't have a positive test case in this scenario. If you think this is _not_ intentional, please open an issue to add a positive test case.